modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -227,18 +227,22 @@ export default class Tile3DLayer extends CompositeLayer {
 
   _makeSimpleMeshLayer(tileHeader, oldLayer) {
     const content = tileHeader.content;
-    const {attributes, modelMatrix, cartographicOrigin, texture} = content;
-    const {positions, normals, texCoords} = attributes;
+    const {modelMatrix, cartographicOrigin, texture} = content;
+    const attributes = {};
```

Comment:
This conversion is not used if `oldLayer.props.mesh` already exists, see below.

Reply:
Exactly like in existing code on `master` branch

Reply:
No. Existing code on master is not executed if `oldLayer` is supplied.

Reply:
Done